pages.ko/common/bash.md

```diff
@@ -1,32 +1,33 @@
 # bash
 
-> Bourne-Again SHell. `sh`-호환 명령 행 인터프리터.
-> 더 많은 정보: <https://gnu.org/software/bash/>.
+> Bourne-Again SHell, an `sh`- 호환 명령 행 인터프리터.
+> 참조 : `zsh`, `histexpand` (history expansion).
+> 더 많은 정보는 여기서 찾아볼 수 있습니다: <https://gnu.org/software/bash/>.
```

Comment:
```suggestion
> 더 많은 정보:: <https://gnu.org/software/bash/>.
```
This is the standard [more information link](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py) we follow across all Korean pages.

Reply:
mine is a more polite and descriptive way. but, if you want. will follow :)

Reply:
> mine is a more polite and descriptive way. but, if you want. will follow :)

Thanks